SPECS/librdkafka/librdkafka.spec

```diff
@@ -0,0 +1,110 @@
+%global debug_package %{nil}
+%define soname 1
+%define _source_payload w9.gzdio
+%define _binary_payload w9.gzdio
+Summary:        The Apache Kafka C library
+Name:           librdkafka
+Version:        1.4.0
+Release:        1%{?dist}
+License:        BSD-2-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Libraries/C and C++
+URL:            https://github.com/edenhill/librdkafka
+Source0:        librdkafka-%{version}.tar.gz
```

Comment:
Please provide a link to the source through a `#Source0: https://...` comment above this line and replace "librdkafka" with "%{name}".

Reply:
Resolved